akka-cluster-tools/src/multi-jvm/scala/akka/cluster/pubsub/DistributedPubSubMediatorSpec.scala

```diff
@@ -29,6 +29,7 @@ object DistributedPubSubMediatorSpec extends MultiNodeConfig {
   commonConfig(ConfigFactory.parseString("""
     akka.loglevel = INFO
     akka.actor.provider = "cluster"
+    akka.actor.serialize-messages = off
```

Comment:
Multi-jvm spec so should exercise the messages that needs to go over wire already